plugins/org.python.pydev.ast/src/org/python/pydev/ast/codecompletion/revisited/visitors/AssignDefinition.java

```diff
@@ -46,6 +46,8 @@
      */
     public final exprType nodeValue;
 
+    public final exprType nodeType;
```

Comment:
Since the `type` is on the `Definition`, this should be there too and not only on `AssignDefinition`.

Reply:
This is still missing.

Reply:
You mean `exprType nodeType` itself or a String containing the type (like the value is set)? I did not get it...

I just added a String type
https://github.com/fabioz/Pydev/blob/89c84733ca9c64f2a35d5d226eadda41c59ba744/plugins/org.python.pydev.ast/src/org/python/pydev/ast/codecompletion/revisited/visitors/Definition.java#L69

Reply:
Yes, I mean that the `public final exprType nodeType;` should be in the `Definition` and not on `AssignDefinition`.

Reply:
Ok, I'll remove it from AssignDefinition then